rds-postgres/admin-login/main.tf

```diff
@@ -79,4 +81,15 @@ data "aws_db_instance" "this" {
 
 locals {
   full_name = join("-", ["rds-postgres", var.identifier])
+
+  # base_value = {
+  #   dbname       = var.database_name
+  #   engine       = data.aws_db_instance.this.engine
+  #   host         = data.aws_db_instance.this.address
+  #   password     = var.initial_password
+  #   port         = tostring(data.aws_db_instance.this.port)
+  #   username     = var.username
+  # }
+
+  # initial_secret_value = can(var.replica_host) ? merge(local.base_value, { replica_host = var.replica_host }) : local.base_value
```

Comment:
Removed so that we're passing the replica DB host address in only one way; as an environment variable rather than also as part of the secrets.